libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -672,6 +672,30 @@ void IRGeneratorForStatements::endVisit(Return const& _return)
 bool IRGeneratorForStatements::visit(UnaryOperation const& _unaryOperation)
 {
 	setLocation(_unaryOperation);
+
+	FunctionDefinition const* function = *_unaryOperation.annotation().userDefinedFunction;
+	if (function)
+	{
+		_unaryOperation.subExpression().accept(*this);
+		setLocation(_unaryOperation);
+
+		solAssert(function->isImplemented());
+		solAssert(function->isFree());
+		solAssert(function->parameters().size() == 1);
+		solAssert(function->returnParameters().size() == 1);
+		solAssert(*function->returnParameters()[0]->type() == *_unaryOperation.annotation().type);
+
+		string argument = expressionAsType(_unaryOperation.subExpression(), *function->parameters()[0]->type());
```

Comment:
```suggestion
		string argument = expressionAsCleanedType(_unaryOperation.subExpression(), *function->parameters()[0]->type());
```

Reply:
Left as is. See https://github.com/ethereum/solidity/pull/13790#discussion_r1113626349.